.drone.yml

```diff
@@ -0,0 +1,20 @@
+kind: pipeline
+name: default
+
+platform:
+  os: linux
+  arch: arm
```

Comment:
Note: If this is changed to `arm64` we get what appears be like #945

https://cloud.drone.io/planck-repl/planck/20/1/2

Reply:
Interesting. So it is a problem with the 64-bit version. That's why I'm not seeing it on my Pi.

Reply:
@pyrmont The 64-bit ARM box in AWS works fine. Its `uname -a` output is:

```
Linux ip-172-31-90-138 4.15.0-1028-aws #29+nutmeg8-Ubuntu SMP Tue Nov 20 02:59:41 UTC 2018 aarch64 aarch64 aarch64 GNU/Linux
```

The failing Drone.io CI box has `uname -a`:

```
Linux b993b3627483 4.15.0-36-generic #39-Ubuntu SMP Wed Sep 26 06:05:44 UTC 2018 aarch64 GNU/Linux
```

I'll probably have a chance at getting to the bottom of it once I get access to the Scaleway ARM environments.

Reply:
Oh yes, sorry. I forgot you'd already got it working on 64-bit. Quite strange.